src/views/CreateEvent.vue

```diff
@@ -1,14 +1,141 @@
 <template>
   <div class="container">
     <h1>This is Create Event page</h1>
+    <form>        
+        <div class="input__wrapper">
+          <label>Event ID: </label>
+          <input type="text" v-model="item.id">
+        </div>
+        <div class="input__wrapper">
+          <label>Title name: </label>
+          <input type="text" v-model="item.title">
+        </div>
+        <div class="input__wrapper">
+          <label>Date: </label>
+          <input type="text" v-model="item.date">
+        </div>
+        <div class="input__wrapper">
+          <label>Time: </label>
+          <input type="text" v-model="item.time">
+        </div>
+        <div class="input__wrapper">
+          <label>Location: </label>
+          <input type="text" v-model="item.location">
+        </div>
+        <div class="input__wrapper">
+          <label>Description: </label>
+          <input type="text" v-model="item.description">
+        </div>
+        <div class="input__wrapper">
+          <label>Organizer: </label>
+          <input type="text" v-model="item.organizer">
+        </div>
+        <div class="input__wrapper">
+          <label>Category: </label>
+          <input type="text" v-model="item.category">
+        </div>
+        <p class="attendees__title">Attendees:</p>                                          
+        <div class="attendees__wrapper" v-for="subitem of item.attendees" :key="subitem.id">
+          <label>ID: </label>
+          <input type="text" v-model="subitem.id">
+          <label>name: </label>
+          <input type="text" v-model="subitem.name">                              
+        </div>                        
+        <div class="attendees__btn" @click="addNewAttendees">Add new Attendees</div>
+     </form>            
+     <button @click="addItem(item)">Save Event</button>     
   </div>
 </template>
 
+<script>  
+
+  import {mapMutations} from 'vuex'
+
+  export default {
+    
+    data() {
+      return {      
+        id: '',
+        item: {                    
+          attendees: []          
+          },        
+        }        
+      },
+    components: {
+      
+    },
+    methods: {
+          ...mapMutations (['ADD_ITEM']),
+          addItem(item) {
+            this.$store.commit('ADD_ITEM', item);
```

Comment:
You already added `mapMutations` helper, so why do you point to the global $store object?